zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java

```diff
@@ -309,6 +309,28 @@ public void testAutoRestartInterpreterAfterSchedule() throws InterruptedExceptio
   }
 
   @Test
+  public void testExportAndImportNote() throws IOException, CloneNotSupportedException,
+          InterruptedException {
+    Note note = notebook.createNote();
+    note.getNoteReplLoader().setInterpreters(factory.getDefaultInterpreterSettingList());
+
+    final Paragraph p = note.addParagraph();
+    String simpleText = "hello world";
+    p.setText(simpleText);
```

Comment:
@Leemoonsoo I cannot agree that's the best way to do that.. 
What if we change type of Job.result into InterpreterResult ?